diff --git a/CMakeLists.txt b/CMakeLists.txt
index de33290c8..d594a62a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -340,7 +340,7 @@ include(ECMQtDeclareLoggingCategory)
 
 ####################################### Program Info #######################################
 
-set(Launcher_APP_BINARY_NAME "prismlauncher" CACHE STRING "Name of the Launcher binary")
+set(Launcher_APP_BINARY_NAME "spectrumlauncher" CACHE STRING "Name of the Launcher binary")
 add_subdirectory(program_info)
 
 ####################################### Install layout #######################################
diff --git a/program_info/CMakeLists.txt b/program_info/CMakeLists.txt
index a4c56e1a0..1edd15fa8 100644
--- a/program_info/CMakeLists.txt
+++ b/program_info/CMakeLists.txt
@@ -8,21 +8,22 @@ if(UNIX)
     endif()
 endif()
 
-set(Launcher_CommonName "PrismLauncher")
-set(Launcher_DisplayName "Prism Launcher")
+set(Launcher_CommonName "SpectrumLauncher")
+set(Launcher_DisplayName "Spectrum Launcher")
+set(Launcher_BinaryName "spectrumlauncher")
 
 set(Launcher_Name "${Launcher_CommonName}" PARENT_SCOPE)
 set(Launcher_DisplayName "${Launcher_DisplayName}" PARENT_SCOPE)
 
-set(Launcher_AppID "org.prismlauncher.PrismLauncher")
+set(Launcher_AppID "org.blocovermelho.SpectrumLauncher")
 set(Launcher_SVGFileName "${Launcher_AppID}.svg")
 set(Launcher_Copyright "© 2022-2026 Prism Launcher Contributors\\n© 2021-2022 PolyMC Contributors\\n© 2012-2021 MultiMC Contributors")
 set(Launcher_Copyright_Mac "© 2022-2026 Prism Launcher Contributors, © 2021-2022 PolyMC Contributors and © 2012-2021 MultiMC Contributors" PARENT_SCOPE)
 set(Launcher_Copyright "${Launcher_Copyright}" PARENT_SCOPE)
-set(Launcher_Domain "prismlauncher.org" PARENT_SCOPE)
+set(Launcher_Domain "launcher.blocovermelho.org" PARENT_SCOPE)
 set(Launcher_UserAgent "${Launcher_CommonName}/${Launcher_VERSION_NAME}" PARENT_SCOPE)
-set(Launcher_ConfigFile "prismlauncher.cfg" PARENT_SCOPE)
-set(Launcher_Git "https://github.com/PrismLauncher/PrismLauncher" PARENT_SCOPE)
+set(Launcher_ConfigFile "${Launcher_BinaryName}.cfg" PARENT_SCOPE)
+set(Launcher_Git "https://github.com/blocovermelho/spectrumlauncher" PARENT_SCOPE)
 set(Launcher_AppID "${Launcher_AppID}" PARENT_SCOPE)
 set(Launcher_SVGFileName "${Launcher_SVGFileName}" PARENT_SCOPE)
 
@@ -31,9 +32,9 @@ set(Launcher_mrpack_MIMEInfo "program_info/modrinth-mrpack-mime.xml" PARENT_SCOP
 set(Launcher_MetaInfo "program_info/${Launcher_AppID}.metainfo.xml" PARENT_SCOPE)
 set(Launcher_PNG_256 "program_info/${Launcher_AppID}_256.png" PARENT_SCOPE)
 set(Launcher_SVG "program_info/${Launcher_SVGFileName}" PARENT_SCOPE)
-set(Launcher_Branding_ICNS "program_info/prismlauncher.icns" PARENT_SCOPE)
-set(Launcher_Branding_MAC_ICON "program_info/PrismLauncher.icon" PARENT_SCOPE)
-set(Launcher_Branding_ICO "program_info/prismlauncher.ico")
+set(Launcher_Branding_ICNS "program_info/${Launcher_BinaryName}.icns" PARENT_SCOPE)
+set(Launcher_Branding_MAC_ICON "program_info/${Launcher_CommonName}.icon" PARENT_SCOPE)
+set(Launcher_Branding_ICO "program_info/${Launcher_BinaryName}.ico")
 set(Launcher_Branding_ICO "${Launcher_Branding_ICO}" PARENT_SCOPE)
 set(Launcher_Branding_WindowsRC "program_info/prismlauncher.rc" PARENT_SCOPE)
 set(Launcher_Branding_LogoQRC "program_info/prismlauncher.qrc" PARENT_SCOPE)
@@ -75,13 +76,13 @@ endif()
 configure_file(win_install.nsi.in win_install.nsi @ONLY)
 
 if(SCDOC_FOUND)
-    set(in_scd "${CMAKE_CURRENT_SOURCE_DIR}/prismlauncher.6.scd")
-    set(out_man "${CMAKE_CURRENT_BINARY_DIR}/prismlauncher.6")
+    set(in_scd "${CMAKE_CURRENT_SOURCE_DIR}/${Launcher_BinaryName}.6.scd")
+    set(out_man "${CMAKE_CURRENT_BINARY_DIR}/${Launcher_BinaryName}.6")
     add_custom_command(
         DEPENDS "${in_scd}"
         OUTPUT "${out_man}"
         COMMAND ${SCDOC_SCDOC} < "${in_scd}" > "${out_man}"
     )
     add_custom_target(man ALL DEPENDS ${out_man})
-    set(Launcher_ManPage "program_info/prismlauncher.6" PARENT_SCOPE)
+    set(Launcher_ManPage "program_info/${Launcher_BinaryName}.6" PARENT_SCOPE)
 endif()
